javascript/javascript3/week2/exercise 1/BadMovies/BadMovies.js

```diff
@@ -0,0 +1,31 @@
+document.body.style.backgroundColor = "skyBlue";
+document.body.style.textAlign = "center";
+document.body.style.listStyle = "none";
+
+
+const myH2 = document.createElement("h2");
+myH2.innerHTML = "Bad Movies";
+myH2.style.color = "brown";
+document.body.appendChild(myH2);
+
+
+
+
+
+
+const movies = "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+
+fetch(movies)
+    .then(Response => Response.json())
+    .then(movies => {
+        const badMovies = movies.filter(movie => movie.rating <= 3);
+        badMovies.forEach(movies => {
+
+            const li = document.createElement("li");
```

Comment:
I thnink, Its good to create class and use in style.css for styling html element. .